.travis.yml

```diff
@@ -0,0 +1,22 @@
+sudo: false
+language: perl
+perl:
+    - "5.24"
+    - "5.22"
+    - "5.18"
+    - "5.14"
+matrix:
+    fast_finish: true
+before_install:
+    - git clone git://github.com/travis-perl/helpers ~/travis-perl-helpers
+    - source ~/travis-perl-helpers/init
+    - build-perl
+    - perl -V
+    - "cpanm -n Moose"
+    - "cpanm -n List::Util"
+    - "cpanm -n Scalar::Util"
+    - "cpanm -n Time::Duration::Concise"
+    - "cpanm -n MooseX::Types::Moose"
+    - "cpanm -n Text::CSV"
```

Comment:
No need of manually install dependencies, including heavy ones like Moose.

See

https://github.com/binary-com/perl-TimeSeries-AdaptiveFilter/blob/master/.travis.yml 

as reference for our open-source module